package.json

```diff
@@ -184,6 +184,7 @@
     "nock": "8.0.0",
     "nodemon": "1.4.1",
     "react-addons-test-utils": "15.4.0",
+    "react-codemod": "github:reactjs/react-codemod#d6563d9",
```

Comment:
@gwwar Unfortunately, react-codemod isn't published on `npm` nor are releases properly tagged on their git repo, so I'll go ahead and shorten this dependency to your suggestion!